middleware/passport.js

```diff
@@ -80,6 +80,9 @@ module.exports = function() {
     }
 
     const makeUserObject = user => {
+        if (!user) {
+            return null;
```

Comment:
We need this because the `user` object we're working with comes from `findById()`, which returns `null` if the user doesn't exist (eg. doesn't trigger any of our `catch()` blocks).

Reply:
... otherwise, this threw an actual (uncaught) error and the user sees an unstyled 500 page for every URL on the site.

Reply:
Mind adding this rationale as a comment to the code?

Reply:
Done!